core/src/test/java/org/apache/calcite/test/TypeCoercionConverterTest.java

```diff
@@ -133,4 +172,24 @@
   private void checkPlanEquals(String sql) {
     tester.assertConvertsTo(sql, "${plan}");
   }
+
+  private void checkValidateSqlEquals(String sql) {
+    Objects.requireNonNull(sql, "sql");
+    final SqlNode sqlQuery;
+    try {
+      sqlQuery = tester.parseQuery(sql);
+    } catch (RuntimeException | Error e) {
+      throw e;
+    } catch (Exception e) {
+      throw TestUtil.rethrow(e);
```

Comment:
Consider using `catch(Throwable e) { TestUtil.rethrow(e, "sql=" + sql) }`
That would be shorter, it would catch all throwables, and it would add the SQL in question

Reply:
I have modified it, thank you very much for your advice.